security/guard_authentication.rst

```diff
@@ -172,8 +172,8 @@ This requires you to implement seven methods::
         public function getCredentials(Request $request)
         {
             if (!$token = $request->headers->get('X-AUTH-TOKEN')) {
-                // No token?
-                $token = null;
+                // No token? Return null and show "Authentication Required"
```

Comment:
Hello @DaumantasUrb, thank you for this first contribution.
Sadly this change is wrong, another guard can still authenticate this request is this one returning `null`. If one want to get the exception at this point an exception should be thrown.

Should you we improve the docs so this is better explained?